shared/css/color.scss

```diff
@@ -1,9 +1,11 @@
 // colors from https://www.dropbox.com/s/ycjiwt8vxlmv2ih/Code.org_stylesheet_v2_09.05.13.pdf
 
+$black: #000000;
 $charcoal: #5b6770;
 $light_gray: #949ca2;
 $lighter_gray: #c6cacd;
 $lightest_gray: #e7e8ea;
+$white: #ffffff;
```

Comment:
Figured we could use a pure black and pure white as well.  Apparently (says hound) good form is to always use variables for colors, even ones that have browser definitions.